src/main/java/net/minecraftforge/fml/common/discovery/ModCandidate.java

```diff
@@ -102,10 +104,12 @@ public boolean isClasspath()
     {
         return classpath;
     }
+    @Deprecated // remove in 1.13
     public void rememberBaseModType(String className)
     {
         baseModTypes.add(className);
```

Comment:
Honestly these are only used in the code you removed, so... screw any modder who calls them.